resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb

```diff
@@ -66,6 +66,8 @@ start() {
     # Move any heap dumps aside
     echo -n $"Starting $prog: "
 
+    [ -z "$OPEN_FILE_LIMIT" ] && ulimit -n $OPEN_FILE_LIMIT
```

Comment:
So we don't want to allow someone to configure these themselves from the defaults, unless we've specifically set a default?